### PR TITLE
Implementation of Cookie policy view

### PR DESCRIPTION
**Pull request by BirajPaul**
**What was done** : Implementation of cookie policy view. The npm package _react-cookie_ was installed to provide cookie related services in the code. The cookie is set for 10 years, which can be changed if necessary. 

**Additional changes** : Fonts were copied from the _Cultural Events_ project and made available in this project. Additionally a few console errors were fixed.

Short preview of the view below as a gif :

cookies

### CODE REVIEW

File: front-end/src/components/cookie/CookieView.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+class CookieView extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            show: true
+        };
+    
+        this.handleClose = this.handleClose.bind(this);
```
Comment (StormBlade97) [inline front-end/src/components/cookie/CookieView.js]: you can autobind with this syntax

method = (argList) => {...}

(declaring a method as a property of the class will make it static and bind it to the class itself)

File: front-end/src/components/cookie/CookieView.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        const cookiePolicy = this.cookies.get('policy');
+        const showCookieDiv = !this.isCookieSet(cookiePolicy) && this.state.show;
+
+        return (
+            <Modal show={showCookieDiv} onHide={this.handleClose} bsClass="cookie-modal">
+                <Modal.Header closeButton>
+                    <Modal.Title>Cookie Policy</Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    The Vantaa Feed website uses cookies to ensure you receive the best experience on our awesome site!
```
Comment (StormBlade97) [inline front-end/src/components/cookie/CookieView.js]: i18n?

File: front-end/src/components/cookie/CookieView.scss [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+.cookie-modal-backdrop {
+    @extend %common-position-rule;
+
+    z-index: 1040;
+    background-color: rgba(74, 74, 74, 0.7);
+}
+
+.cookie-modal {
```
Comment (StormBlade97) [inline front-end/src/components/cookie/CookieView.scss]: There is a modal implementation from the previous project. You might want to reuse it

Comment (StormBlade97) [reply front-end/src/components/cookie/CookieView.scss]: It is here:
https://github.com/City-of-Vantaa-SmartLab/CultureEvents/blob/master/frontend/src/components/modal/index.js

File: front-end/src/components/cookie/CookieView.js (hunk 1 above)
Comment (BirajPaul) [reply front-end/src/components/cookie/CookieView.js]: Thanks, will check it.

File: front-end/src/components/cookie/CookieView.js (hunk 2 above)
Comment (BirajPaul) [reply front-end/src/components/cookie/CookieView.js]: Yes, I think internationalization should be done across the whole project as a separate ticket. I will create one for that.

File: front-end/src/components/cookie/CookieView.scss (hunk 3 above)
Comment (BirajPaul) [reply front-end/src/components/cookie/CookieView.scss]: Thanks, will check it.

Comment (BirajPaul) [reply front-end/src/components/cookie/CookieView.scss]: As we discussed, we can go with Bootstrap modal as is used here.

**Pull request merged**